using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -88,19 +88,31 @@ Number of seconds to wait before Analytics launch hits are sent from the SDK. En
 
 {% tabs %}
 {% tab title="Android" %}
-Add the Analytics extension to your project using the app's Gradle file.
+1. Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) and Analytics extensions to your project using the app's Gradle file.
+
+```java
+implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+implementation 'com.adobe.marketing.mobile:analytics:1.+'
+```
 
 #### Java
 
-Import the Analytics extension in your application's main activity.
+2. Import the Analytics extension in your application's main activity.
 
 ```java
 import com.adobe.marketing.mobile.*;
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-​ 1. Add the library to your project via your Cocoapods `Podfile` by adding `pod 'ACPAnalytics'` ​ 2. Import the Analytics and Identity libraries:
+{% tab title="iOS" %}​
+1. Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) and Analytics extensions to your project using Cocoapods. Add following pods in your `Podfile`  
```

Comment:
- the 2nd line sounds like a step...please move to a new line and add a "2."
- Please add a colon (:) after `Podfile`: